Cargo.toml

```diff
@@ -21,7 +21,7 @@ license = "ISC"
 name = "rustls-webpki"
 readme = "README.md"
 repository = "https://github.com/rustls/webpki"
-version = "0.102.6"
+version = "0.102.7"
```

Comment:
If possible I'd like to hold publishing this release until https://github.com/rustls/webpki/pull/276 lands. I suspect it will be in a matter of days at most :crossed_fingers:

Reply:
276 hit `main` so this consideration isn't applicable anymore